tests/integration/helpers/helpers.py

```diff
@@ -232,10 +231,16 @@ def relation_exited(ops_test: OpsTest, endpoint_one: str, endpoint_two: str) ->
     return False
 
 
-async def deploy_postgres_bundle(ops_test: OpsTest, scale_postgres: int = 2, timeout=(60 * 10)):
+async def deploy_postgres_bundle(
+    ops_test: OpsTest, scale_postgres: int = 2, timeout: int = (60 * 10), focal: bool = False
+):
     """Deploy postgresql bundle."""
     async with ops_test.fast_forward():
         await ops_test.model.deploy("./releases/latest/postgresql-bundle.yaml")
+        if focal:
+            await ops_test.model.remove_application(PGB, block_until_done=True)
+            await ops_test.model.deploy(PGB, channel="1/edge", series="focal", num_units=0)
+            await ops_test.model.relate(PG, PGB)
```

Comment:
Tried using an overlay, but Juju CLI needs an explicit revision, while lib juju just pulls the latest revision.